sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -268,7 +268,6 @@ def set_authorizer(cls, authorizer: str):
     developer_mode: bool = None  # False
     skip_wait_for_gossip_to_settle: int = None  # -1
     force_gossip_generation: int = None  # -1
-    experimental: bool = None  # False
```

Comment:
if it's removed from here, `experimental_feature` should be added instead

Reply:
see the next line. it's already there.